django/db/backends/sqlite3/features.py

```diff
@@ -44,6 +44,14 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_order_by_nulls_modifier = Database.sqlite_version_info >= (3, 30, 0)
     order_by_nulls_first = True
 
+    @cached_property
+    def supports_atomic_references_rename(self):
+        # Sqlite 3.28.0 bundled with MacOS 10.15 does not support renaming references atomically.
+        if platform.mac_ver()[0].startswith("10.15.") and Database.sqlite_version_info == (3, 28, 0):
```

Comment:
```suggestion
        if platform.mac_ver()[0].startswith('10.15.') and Database.sqlite_version_info <= (3, 28, 0):
```
What about 3.26.X and 3.27.X?

Reply:
I think the point is that it's specifically this version that is bundled with macOS 10.15. (I'm on 10.14 and don't see it.)